supabase/migrations/20240422113025_allow_null_postcode.sql

```diff
@@ -0,0 +1,5 @@
+alter table "public"."clients" drop constraint "clients_address_postcode_check";
+
+alter table "public"."clients" alter column "address_postcode" drop not null;
+
```

Comment:
As well as dropping the "not null" constraint on the postcode, I also needed to drop a custom constraint where we checked that the length of postcode was greater than 0.